app/controllers/foreman_ansible/concerns/hosts_controller_extensions.rb

```diff
@@ -42,7 +42,7 @@ def multiple_play_roles
           @composer.trigger
           redirect_to job_invocation_path(@composer.job_invocation)
         else
-          #raise ::Foreman::Exception.new(N_("Could not run Ansible roles for %s") % @host)
+          raise ::Foreman::Exception.new(N_("Could not run Ansible roles %s") % @composer.errors)
```

Comment:
this is quite specific exception, should we create new child class of Foreman::Exception for it? something like AnsibleRolePlayError